client/my-sites/upgrades/domain-management/transfer/index.jsx

```diff
@@ -30,45 +30,38 @@ const Transfer = React.createClass( {
 		wapiDomainInfo: React.PropTypes.object.isRequired
 	},
 
+	renderSection() {
+		const { locked, pendingTransfer } = this.props.wapiDomainInfo.data;
+		const { isPendingIcannVerification, transferProhibited } = getSelectedDomain( this.props );
```

Comment:
[There can be only one](https://www.youtube.com/watch?v=sqcLjcSloXs)... `const` declaration, at least per our guidelines.